codice/ddf
# DDF-5581 Cosmetic Changes to Ruler on Map

#### What does this PR do?
The Ruler on Map feature should show the distance between points via the MapInfo component on the bottom right of the screen and next to the mouse. That should also calculate the distance between the start point and the mouse when in the "START" state.

Other changes include:
- Fix to explicitly render points and lines when those functions are called
- Color changes to the line and points
- Removing the toast upon creating a line

#### Who is reviewing it? 

@rfding 
@mdang8 
@aaronilovici 

#### Select relevant component teams: 
@codice/ui 


#### Ask 2 committers to review/merge the PR and tag them here.
@brjeter
@andrewkfiedler

#### How should this be tested?
1. pull and build changes.
2. Open up the Search UI and add a tab for the 3D map and the 2D map.
3. Start with the 3D map. Select a point on the globe, right click and go to **Calculate Distance -> Select Start Point**. 
4. Verify that the distance is being updated in both the bottom left corner and next to the mouse. Zoom in close to the start point and hover the mouse near enough to verify that the measurement format changes to meters when distance < 1 km.
5. Right click and add a stop endpoint. Verify that the distance tooltip goes away but the bottom left still display's the distance.
6. Right click on the map and click **Calculate Distance -> Copy Coordinates**. Check that the copied coordinates are the same as the distance initially displayed.
7. Repeat steps 3-7 again on the same map to make sure the state changes haven't affected the usage.
8. Repeat steps 3-8 again on the 2D map.

#### What are the relevant tickets?
Fixes: #_DDF-5581_

#### Screenshots
![image](https://user-images.githubusercontent.com/11358088/72280200-cb31db00-3605-11ea-9263-141a355a2206.png)
![image](https://user-images.githubusercontent.com/11358088/72280282-fcaaa680-3605-11ea-8c9a-000d3a405362.png)


#### Checklist:


#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
